Remotion/Web/Development.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,5 @@
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|--------------------
+RMWEB0001  | Usage | Error | Wrong WebString or PlainTextString usage.
```

Comment:
Those files are required and lead to compiler errors if missing. It is unshipped because the rule currently is not released. I'm not sure how to handle this, I guess it would be too much work to keep moving entries from one file to the other on release, and I actually don't want to look into an msbuild integration (unless you really want it). I would propose to add them to the shipped file (versioned correctly) directly as they are added, leaving the unshipped file empty.

Reply:
looks good to me (moving it to shipped). Once a rule is done, it's done and that should be at the end of the feature branch anyway. so, for the most part, shipped will be the correct choice from the get go. If we ever have a rule that's done but kind if iffy, we can still keep it in unshipped.